Task50/Tests/TestBaseClass.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Tests;
+
+[TestFixture]
+public class TestBaseClass
+{
+    protected IWebDriver _driver;
+
+    [SetUp]
+    public void Setup()
+    {
+        _driver = new ChromeDriver();
+        _driver.Manage().Window.Maximize();
+    }
+
+    [TearDown]
+    public void CleanUp()
+    {
+        _driver.Quit();
+        _driver.Dispose();
```

Comment:
You don't need to call Dispose method after Quit because Dispose already is called within Quit